ui/src/app/components/toast-banner.tsx

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+
+import {FlexColumn, FlexRow} from './flex';
+import {ClrIcon} from './icons';
+import colors, {colorWithWhiteness} from 'app/styles/colors';
+import {reactStyles} from 'app/utils';
+import {CSSProperties} from 'react';
+
+const styles = reactStyles({
+  banner: {
+    backgroundColor: colorWithWhiteness(colors.accent, .9),
```

Comment:
this differs from the original styling.  The new default of the ToastBanner is a pleasant blue `accent` and does not have a warning icon (for future RW-7060 work).

To enable the status-alert styling, set `warningStyle` to true.